opt_einsum/backends/torch.py

```diff
@@ -31,6 +31,12 @@ def transpose(a, axes):
 def einsum(equation, *operands):
     """Variadic version of torch.einsum to match numpy api.
     """
+    # rename symbols to support PyTorch 0.4.1 and earlier,
+    # which allow only symbols a-z.
+    symbols = sorted(set(equation) - set(',->'))
```

Comment:
Extreme, but can we throw if `len(symbols) > 26`.

Reply:
The advantage of not throwing is that this will continue to work when PyTorch fixes their einsum to allow more symbols. I'd rather let their current implementation throw.

Reply:
Makes sense. I am not intimately familiar with the new `torch` symbols set, but would `opt_einsum.parser.convert_to_valid_einsum_chars` work here?